utils/_context/_scenarios/parametric.py

```diff
@@ -444,39 +444,36 @@ def dotnet_library_factory():
         container_name="dotnet-test-api",
         container_tag="dotnet8_0-test-api",
         container_img=f"""
-FROM mcr.microsoft.com/dotnet/sdk:8.0 as build
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
+WORKDIR /app
 
 # `binutils` is required by 'install_ddtrace.sh' to call 'strings' command
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y curl binutils
-WORKDIR /app
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y binutils
 
-# ensure that the Datadog.Trace.dlls are installed from /binaries
-COPY utils/build/docker/dotnet/install_ddtrace.sh utils/build/docker/dotnet/query-versions.fsx binaries* /binaries/
+COPY utils/build/docker/dotnet/install_ddtrace.sh utils/build/docker/dotnet/query-versions.fsx binaries/ /binaries/
 RUN /binaries/install_ddtrace.sh
 
-# restore nuget packages
+# dotnet restore
 COPY {dotnet_reldir}/ApmTestApi.csproj {dotnet_reldir}/nuget.config ./
 RUN dotnet restore "./ApmTestApi.csproj"
 
-# build and publish
+# dotnet publish
 COPY {dotnet_reldir} ./
-RUN dotnet publish --no-restore --configuration Release --output out
+# RUN dotnet publish --no-restore -c Release -f net8.0 -o out
+RUN dotnet publish --no-restore -c Release -o out
 
 ##################
 
-FROM mcr.microsoft.com/dotnet/aspnet:8.0 as runtime
-COPY --from=build /app/out /app
-COPY --from=build /app/SYSTEM_TESTS_LIBRARY_VERSION /app/SYSTEM_TESTS_LIBRARY_VERSION
-COPY --from=build /opt/datadog /opt/datadog
+FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS runtime
 WORKDIR /app
 
 # Opt-out of .NET SDK CLI telemetry (prevent unexpected http client spans)
 ENV DOTNET_CLI_TELEMETRY_OPTOUT=1
 
 # Set up automatic instrumentation (required for OpenTelemetry tests),
 # but don't enable it globally
-ENV CORECLR_ENABLE_PROFILING=0
-ENV CORECLR_PROFILER={{846F5F1C-F9AE-4B07-969E-05C26BC060D8}}
+ENV CORECLR_ENABLE_PROFILING=1
+ENV CORECLR_PROFILER='{{846F5F1C-F9AE-4B07-969E-05C26BC060D8}}'
```

Comment:
None of the other env vars have quotes. Why add them here?

Reply:
The env var won't have quote, it's interpreted as the syntax of dockerfile : 

![image](https://github.com/user-attachments/assets/b1081a70-777a-405d-b3c1-bda9a1af3a8a)

While not being necessary, I did that because the weblog was fine, and I set this to easily compare the dockerfile of the weblog with the dockerfile of parametric test. And the weblog dockerfile has those quotes.

Reply:
Ok, not a big deal, just curious. Thanks for clarifying.